README.md

```diff
@@ -62,3 +73,21 @@ npm publish
 ```
 
 When you're confident with the release, make sure the version tag is also released at GitHub.
+
+## Support
+
+[Frequently Asked Questions](http://dev.flagpack.xyz/support/)
+
+[Documentation](http://dev.flagpack.xyz/docs/)
+
+## Releases
+
+You can find a changelog of Flagpack's releases on the [Releases page](https://github.com/Yummygum/react-flagpack/releases) on GitHub.
+
+## Contribute
+
+If you're interested in contributing to this project, great! Please see the [contributing document](CONTRIBUTING.md).
+
+## License
+
+Flagpack is an open source project published under a [MIT license](LICENSE).
```

Comment:
I feel like we can give a little shoutout to our [Yummygum website](https://www.yummygum.com/) here at the end. No harm in linking to show who is behind this project ❤️.